scripts/.fonctions

```diff
@@ -1,34 +1,42 @@
 #!/bin/bash
 
-CHECK_VAR () {	# Vérifie que la variable n'est pas vide.
-# $1 = Variable à vérifier
-# $2 = Texte à afficher en cas d'erreur
+ynh_version="2.4"
+
+YNH_VERSION () {	# Returns the version number of the Yunohost moulinette
+	ynh_version=$(sudo yunohost -v | grep "moulinette:" | cut -d' ' -f2 | cut -d'.' -f1,2)
+}
+
```

Comment:
+ynh_version="2.4"
+
+YNH_VERSION () {	# Returns the version number of the Yunohost moulinette
+	ynh_version=$(sudo yunohost -v | grep "moulinette:" | cut -d' ' -f2 | cut -d'.' -f1,2)
+}
Pas besoin ici.